src/main/java/nl/tudelft/semgroup4/MainApp.java

```diff
@@ -61,7 +61,7 @@ public static void main(String[] args) throws SlickException {
     public void initStatesList(GameContainer container) throws SlickException {
         Resources.init();
         Settings.init();
-        addState(new StartScreenState());
+        addState(new StartScreenState(new ResourcesWrapper()));
```

Comment:
That is not the case, because the resources are passed to the state controller immediately. This controller's constructor then tries to create a font with that resourcesWrapper.
In tests this crashes.

Maybe movign the font creation outside of the constructor, but that would allow for a possibly unset field.

< testing, interface